plugins/nsmserver/nsmserver_plugin.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// //go:generate protoc -I ./model/pod --go_out=plugins=grpc:./model/pod ./model/pod/pod.proto
+
+package nsmserver
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ligato/cn-infra/flavors/local"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/networkservicemesh/plugins/objectstore"
+	"github.com/ligato/networkservicemesh/utils/idempotent"
+)
+
+// Plugin watches K8s resources and causes all changes to be reflected in the ETCD
+// data store.
+type Plugin struct {
+	nsmClientEndpoints nsmClientEndpoints
+	Deps
```

Comment:
I *think* you want Deps first here.

Reply:
it does not change any logic, but to be consistent with other plugins, will change it.